deploy/ansible/roles/docker/tasks/clean_docker.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: remove all docker containers
```

Comment:
You should not assume it is safe to clean up all the images and containers on the system. Other containers may also present on the same system.